### DISCUSSION TITLE
BW-SHP13 aka Tuya TS0121_plug aka _TZ3000_g5xawfcq => re-flash to TS011F_plug_1 ?

### DISCUSSION THREAD

**Discussion by Radoom** [Koenkk/zigbee2mqtt#14887] [Devices]
Hi all, 
not sure if this is the right place but since there's a lot of exchange on the variants and (past?) problems with the [TS011F_plug_1](https://www.zigbee2mqtt.io/devices/TS011F_plug_1.html) vs. [TS011F_plug_3](https://www.zigbee2mqtt.io/devices/TS011F_plug_3.html) on the issues/discussion section here, I hope here's the place where the pros are.

I have 9 identical BlitzWolf BW-SHP13 identifying themselves as "TS0121_plug" (without _1 or _3), and "_TZ3000_g5xawfcq". 

I had one of them connected to our oven & microwave for a couple days and used the oven a number of times with it. Yesterday, when the MW was running (at only ~300W), the plug repeatedly switched of suddenly after a couple of seconds or a couple of 10s of seconds, this switching off was reproducible, the timing seemed random, not clearly attributable to a particular moment after starting the MW, just "suddenly some time". 

Running the oven on higher loads the days before nothing ever happened, so it shouldn't be an "overcurrent/voltage-trip" (at least not one caused by the load of the MW itself). So I suspect some interference of the MW radiation (eh, it shouldn't really radiate, should it!? anyhow...) with the plug, as other's had apparently, too, but not sure with what model.

Haven't tried other plugs in the same place yet, nor other "arrangements" in that cupboard, and also don't have any logs of current/voltage for that situation (yet; would need to set things up for that). Having read about the different behaviors/issues with the various models of this otherwise (for now) working good plug(s) I wonder:
Can the firmware on this "TS0121_plug" be upgraded to the later TS011F_Plug(_1)? Anyone done that?
And would it maybe help to prevent such trips (and potentially solve other issues I've not yet spotted as I've not tried much in terms of data recording with them yet). What version / firmware file would be the one to go with?

I'd have to do it in my ioBroker installation in the end, will find out how - but only if it's possible/advisable at all?

Any thoughts on this? Cheers!

**Comment by c0mplex1** [comment]
I can't really answer your question, but I use 2 BlitzWolf BW-SHP13 for almost 2 years and have no issues with them. They work reliable.
Both my BlitzWolf identify themselves as TuYa  TS0121_plug  _TZ3000_g5xawfcq
One is connected with a dishwasher (2200W) and the other one with a close-in boiler (2200W)
I don't have the possibility to upgrade the firmware, because this is not available.

**Comment by LHZ0316** [comment]
来信已收到。
